From 1e91df27a378937168e620993236cfd4d4b5a56d Mon Sep 17 00:00:00 2001
From: Zhang Jiejing <jiejing.zhang@freescale.com>
Date: Mon, 20 Feb 2012 13:00:40 +0800
Subject: ENGR00174829-1 MX6Q_SABRESD: fix warnning message

Fix below warnning message:

arch/arm/mach-mx6/board-mx6q_sabresd.c:753:
warning: 'mx6q_sabresd_flexcan_gpios' defined but not used

Signed-off-by: Zhang Jiejing <jiejing.zhang@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabresd.c |    5 -----
 1 file changed, 5 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabresd.c b/arch/arm/mach-mx6/board-mx6q_sabresd.c
index 55290c2..e859521 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabresd.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabresd.c
@@ -750,11 +750,6 @@ static struct ahci_platform_data mx6q_sabresd_sata_data = {
 	.exit = mx6q_sabresd_sata_exit,
 };
 
-static struct gpio mx6q_sabresd_flexcan_gpios[] = {
-	{ MX6Q_SABRESD_CAN1_EN, GPIOF_OUT_INIT_LOW, "flexcan1-en" },
-	{ MX6Q_SABRESD_CAN1_STBY, GPIOF_OUT_INIT_LOW, "flexcan1-stby" },
-};
-
 static void mx6q_sabresd_flexcan0_switch(int enable)
 {
 	if (enable) {
-- 
1.7.9.5

